jobs/integration/conftest.py

```diff
@@ -399,10 +399,11 @@ def log_dir(request):
     return path
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="session")
 def event_loop():
     loop = asyncio.new_event_loop()
     yield loop
+    loop.close()
```

Comment:
overrides the fixture in `pytest-asyncio:event_loop` which is a "function" based scope to instead be session based so that modules based fixtures can be `async/await`